test/test_config.py

```diff
@@ -30,7 +30,15 @@ def test_read_socket_from_environment_variables():
     os.environ['SPIFFE_ENDPOINT_SOCKET'] = fake_socket
 
     setter = ConfigSetter()
-    del os.environ['SPIFFE_ENDPOINT_SOCKET']
```

Comment:
Does the environment variable still get deleted at the end of testing? Just want to make sure environment state is reset at the end is all. Thanks!

Reply:
yes, there's a post step for this suite that restores the environment state after each test run: https://github.com/HewlettPackard/py-spiffe/pull/33/files#diff-45fa5f8530eb815c0cf606032e587adedb04dc70e59b203535d51c44654cfc4dR7

Reply:
now that you mention this, I think this is a nice util function that we can reuse across test suites

Reply:
Ah thanks, that makes sense why I didn't see it now. Thanks!